lib/irrlicht/source/Irrlicht/CBlit.h

```diff
@@ -513,8 +155,6 @@ static void executeBlit_TextureCopy_x_to_x( const SBlitJob * job )
 	}
 }
 
-/*!
-*/

```

Comment:
No one cares about two lines, it makes it potentially easier to merge with irrlicht updates later (less conflicts) - admittedly unlikely that we will ever do that though ;)